zombie-bite-scripts/helpers.ts

```diff
@@ -67,7 +69,10 @@ export async function monitMigrationFinish(
 
 async function mock_finish(delay_ms: number, is_mocked: boolean) {
   mock_finish_flag = is_mocked;
-  if(is_mocked) await delay(delay_ms);
+  if(is_mocked) {
+    logger?.debug('Mock finish enabled, waiting', { delay_ms });
```

Comment:
```suggestion
    logger.debug('Mock finish enabled, waiting', { delay_ms });
```
The imported logger module should exist.